### PR TITLE
Tator Tot Fix

### PR DESCRIPTION
**Pull request by Niknakflak**
Potato wedges are no longer invisible.
However, wedges still give tator tots instead of french fries and I don't know why and when I asked for help, I was told to figure it the fuck out.  It remains broken and I still, have no idea why.

### CODE REVIEW

File: code/modules/food_and_drinks/kitchen_machinery/processor.dm [hunk 1]
```diff
@@ -102,7 +102,7 @@
 	input = /obj/item/weapon/reagent_containers/food/snacks/grown/potato
 	output = /obj/item/weapon/reagent_containers/food/snacks/tatortot
 
-/datum/food_processor_process/potatowedges
+/datum/food_processor_process/wedges
```
Comment (Niknakflak) [inline code/modules/food_and_drinks/kitchen_machinery/processor.dm]: The culprit here, doesn't output fries despite the output being set to fries

File: code/modules/hydroponics/grown/potato.dm [hunk 2]
```diff
@@ -30,7 +30,7 @@
 /obj/item/weapon/reagent_containers/food/snacks/grown/potato/wedges
```
Comment (Niknakflak) [inline code/modules/hydroponics/grown/potato.dm]: If that were the case, sweet potatos would have the same problem, which they don't

**Comment by optimumtact** [comment]
Is this a fix or a cry for help

**Comment by Niknakflak** [comment]
It's a passive aggressive jab at your attitude

File: code/modules/hydroponics/grown/potato.dm (hunk 2 above)
Comment (Niknakflak) [inline code/modules/hydroponics/grown/potato.dm]: which is what, putting wedges above the potatos?  Then won't it turn into a sweet potato?

**Comment by optimumtact** [comment]
Okay

**Pull request closed without merging**